gef.py

```diff
@@ -7536,8 +7553,10 @@ def _hexdump(self, start_addr, length, arrange_as):
         fmt_pack = endianness + r
         lines = []
 
-        i = 0
-        while i < length:
+        i = 0 + self.repeat_count * length
```

Comment:
Why not calculate the start address _outside_ of this function (e.g. near where you do the above `fmt == "byte"`. That way this function doesn't need `self` at all.

Reply:
I wanted to keep the offsets valid, if we change `start_addr` then they reset to 0 on repeat. Could add a new param to this function instead though?

Reply:
Ah! Yes fair point. That might make sensei `offset=None`, and if it's set, use it as the offset, otherwise use `start_address`

Reply:
`start_offset` or `base`, your call